audit-connector/src/main/java/com/sequenceiq/cloudbreak/audit/converter/AuditEventDataUtilizer.java

```diff
@@ -0,0 +1,12 @@
+package com.sequenceiq.cloudbreak.audit.converter;
+
+import com.cloudera.thunderhead.service.audit.AuditProto;
+import com.sequenceiq.cloudbreak.audit.model.ResultEventData;
+
+public interface AuditEventDataUtilizer<D extends ResultEventData> {
+
+    void modifyAuditEventBuilderWithEventData(AuditProto.AttemptAuditEventResult.Builder auditEventBuilder, D source);
```

Comment:
modifyAuditEventBuilderWithEventData -> convertInBuilder

auditEventBuilder -> attemptAuditEventResultBuilder (too long, I know. or just `builder`)

Reply:
builder will be fine